### PR TITLE
Feat/log interest

### PR DESCRIPTION
**Pull request by Delfil**
Fixing interest bar & removing unused interesting classes

### CODE REVIEW

**Comment by kwendel** [comment]
looks good!

File: src/main/java/tudelft/ti2806/pl3/data/wrapper/util/interest/ComputeInterest.java [hunk 1]
```diff
@@ -8,13 +8,13 @@
 public final class ComputeInterest {
 	private ComputeInterest() {
 	}
-
+	
 	public static void compute(List<Wrapper> wrappers) {
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/data/wrapper/util/interest/ComputeInterest.java]: *Auto-generated comment*

    Missing a Javadoc comment.

**Comment by TomBrouws** [comment]
Looks good except for Javadoc!

**Comment by mathieupost** [comment]
Looks nice!

**Pull request merged**